models/subventions/association/index.ts

```diff
@@ -26,24 +26,34 @@ export const getSubventionsAssociationFromSlug = async (
     isBot: false,
   });
 
-  const { siren } = uniteLegale;
-
+  const {
+    siren,
+    association: { idAssociation },
+  } = uniteLegale;
   try {
-    return await clientApiDataSubvention(siren);
+    return await clientApiDataSubvention(idAssociation || siren);
   } catch (e: any) {
     if (e instanceof HttpNotFound) {
       return APINotRespondingFactory(EAdministration.DJEPVA, 404);
+    } else {
+      try {
+        return await clientApiDataSubvention(siren);
+      } catch (e: any) {
+        if (e instanceof HttpNotFound) {
+          return APINotRespondingFactory(EAdministration.DJEPVA, 404);
+        }
+        logErrorInSentry(
+          new FetchRessourceException({
+            ressource: 'DataSubvention',
+            cause: e,
+            context: {
+              siren,
+            },
+            administration: EAdministration.DJEPVA,
```

Comment:
Sounds great, I see two (minor) improvements : 
- could be interesting to have temporary logs in order to know if this work. A `logInfoInSentry` that says fallback worked for association XXX should be enough 
- current implem will call twice with siren if no IdAssociation. Maybe prefer a first call with siren and then, when it fails : 

```
...
catch(e) {
	if (e instanceof HttpNotFound) {
	          return APINotRespondingFactory(EAdministration.DJEPVA, 404);
        }
	if (idAssociation) {
		try {
			return await clientApiDataSubvention(idAssociation);
		} catch...
	} else {
		return APINotRespondingFactory(EAdministration.DJEPVA, 500);
	}
}
```